### ISSUE TITLE
Intellectual claim on Webojis.com

### ISSUE THREAD

**Issue by creativenoobie** [jeeliz/jeelizWeboji#45]
Hey,

First of all, I am a big fan of your work.

I have some questions related to Apple's claim of intellectual property rights on webojis.com

I want to know if Apple has claimed full intellectual property right on implementation i.e. no similar application (irrespective of technology used) can be developed or just few characters (eg. Foxes, Raccoon etc.) I wanted to develop an open source application but with different emojis so that's why I thought its best to consult you guys first.

I am really pissed off at the fact that they have claimed intellectual property rights on something that was a concept before their "implementation". I really can't believe that.

Looking forward for your reply.

Thank you in advance!

**Comment by xavierjs** [comment]
Hi @creativenoobie 

Apple threatened us (they were kind to not sue us directly) because we had reproduced the fox, we were also describing the weboji as "Anim***i on the web".

So I changed the text, I removed some tutorials on jeeliz.com, I dropped https://webojis.com and I changed the fox to a raccoon.

Here are my tips:
- do not talk about Ani***ji in your project,
- do not reproduce the same animals as Apple.
